src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_commands.py

```diff
@@ -0,0 +1,202 @@
+        client_factory=cf_mysql_flexible_servers
+    )
+
+    mysql_flexible_firewall_rule_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.rdbms.mysql_flexibleservers.operations#FirewallRulesOperations.{}',
+        client_factory=cf_mysql_flexible_firewall_rules
+    )
+
+    mysql_flexible_config_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.rdbms.mysql_flexibleservers.operations#ConfigurationsOperations.{}',
+        client_factory=cf_mysql_flexible_config
+    )
+
+    mysql_flexible_db_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.rdbms.mysql_flexibleservers.operations#DatabasesOperations.{}',
+        client_factory=cf_mysql_flexible_db
+    )
+
+    mysql_flexible_replica_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.rdbms.mysql_flexibleservers.operations#ReplicasOperations.{}',
+        client_factory=cf_mysql_flexible_replica
+    )
+
+    mysql_flexible_location_capabilities_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.rdbms.mysql_flexibleservers.operations#LocationBasedCapabilitiesOperations.{}',
+        client_factory=cf_mysql_flexible_location_capabilities
+    )
+
+    postgres_flexible_servers_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.rdbms.postgresql_flexibleservers.operations#ServersOperations.{}',
+        client_factory=cf_postgres_flexible_servers
+    )
+
+    postgres_flexible_firewall_rule_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.rdbms.postgresql_flexibleservers.operations#FirewallRulesOperations.{}',
+        client_factory=cf_postgres_flexible_firewall_rules
+    )
+
+    postgres_flexible_config_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.rdbms.postgresql_flexibleservers.operations#ConfigurationsOperations.{}',
+        client_factory=cf_postgres_flexible_config
+    )
+
+    postgres_flexible_location_capabilities_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.rdbms..postgresql_flexibleservers.operations#LocationBasedCapabilitiesOperations.{}',
+        client_factory=cf_postgres_flexible_location_capabilities
+    )
+
+    ### MERU COMMANDS
+    flexible_server_custom_common = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.rdbms.flexible_server_custom_common#{}')
+    flexible_servers_custom_postgres = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.rdbms.flexible_server_custom_postgres#{}')
+    flexible_servers_custom_mysql = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.rdbms.flexible_server_custom_mysql#{}')
+
+    ## Postgres commands
+    with self.command_group('postgres flexible-server', postgres_flexible_servers_sdk,
+                            custom_command_type=flexible_servers_custom_postgres,
+                            client_factory=cf_postgres_flexible_servers) as g:
+        g.custom_command('create', '_flexible_server_create', table_transformer=table_transform_output)
```

Comment:
Usually we define custom functions as public (not starts with `_`).

Reply:
Addressed